bundler.d/ovirt.rb

```diff
@@ -1,3 +1,3 @@
 group :ovirt do
-  gem 'rbovirt', '~> 0.0.24'
+  gem "rbovirt", ">= 0.0.31"
```

Comment:
Probably best to keep the `~>` as I think there are some incompatible changes planned for rbovirt.